content/guides/core/reports/edit-a-report.md

```diff
@@ -394,4 +394,11 @@ Select a panel grid and press `delete` on your keyboard to delete a panel grid.
 
 Collapse headers in a Report to hide content within a text block. When the report is loaded, only headers that are expanded will show content. Collapsing headers in reports can help organize your content and prevent excessive data loading. The proceeding gif demonstrates the process.
 
-{{< img src="/images/reports/collapse_headers.gif" alt="" >}}
\ No newline at end of file
+{{< img src="/images/reports/collapse_headers.gif" alt="" >}}
+
+## Visualize relationships across multiple dimensions
```

Comment:
Relationships of what?  Maybe "dimension" means a specific thing here and it is clearer to customers than to me.